.ci/docker/common/install_conda.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```

Comment:
Removing '#' is definitely a typo

Reply:
I'm temp to add this back, but then I realize that you're using ghstack :)

Reply:
Surprisingly it still works...